apps/site/assets/ts/schedule/components/line-diagram/StopCard.tsx

```diff
@@ -91,23 +101,20 @@ const StopCard = (props: StopCardProps): ReactElement<HTMLElement> => {
 
         <div className="m-schedule-diagram__stop-details">
           {StopConnections(routeStop.connections)}
-          {!isDestination &&
-          liveData &&
-          liveData.headsigns.length &&
-          liveData.headsigns.some(hasPredictionTime) ? (
+          {showPrediction ? (
```

Comment:
I think there's already a test for conditionally showing the diversion, is showing the prediction also already covered?

Reply:
Maybe not, will check into it!